portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/JobAuthorization.java

```diff
@@ -0,0 +1,99 @@
+package org.dataportabilityproject.spi.cloud.types;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import org.dataportabilityproject.types.transfer.EntityType;
+
+/**
+ * Authorization data related to a job.
+ */
+public class JobAuthorization extends EntityType {
+
+  /**
+   * The current state of the job.
+   *
+   * <p>The value PENDING_WORKER_ASSIGNMENT indicates the client has sent a request for a worker to
+   * be assigned before sending all the data required for the job.
+   *
+   * <p>The value ASSIGNED_WITHOUT_AUTH_DATA indicates the client has submitted all data required,
+   * such as the encrypted auth data, in order to begin processing the job.
+   */
+  public enum State {
+    // The job is in the process of obtaining source and destination authorization credentials.
+    INITIAL,
+    // The job has all authorization credentials but is not assigned a worker yet
+    PENDING_WORKER_ASSIGNMENT,
+    // The job is assigned a worker instance, authorization credentials have not been populated.
+    ASSIGNED_WITHOUT_AUTH_DATA,
+    // The job is assigned a worker and has encrypted authorization credentials populated.
+    ASSIGNED_WITH_AUTH_DATA,
+  }
+
+  @JsonProperty
+  private JobAuthorization.State state;
+
+  @JsonProperty
+  private String encryptedSourceAuthData;
```

Comment:
Looks like we call it import/export vs source/dest elsewhere, can you stick to that terminology if that's the case?

Reply:
Done